news/2020-11-29-a-labour-of-love.md

```diff
@@ -0,0 +1,103 @@
+
+With teams being public, we'll be taking a different approach from usual contests: **entries will NOT be anonymized.**
+
+In fact, we encourage you to tease your submission as it's being created! Feel free to announce your team publicly and show off your work. We're celebrating an osu! classic and we can't very well do that if everyone keeps their work hidden.
+
+*"I'm a super cool mapper/storyboarder, but I don't know anyone who can remix songs. What do I do?"* There's likely a lot of people in this situation. We recommend anyone looking for teammates to post in the comments saying so. Forming a suitable team is half of the challenge for this contest!
+
+## Schedule
+
+- **3 months:** Contest is open for submissions [here](https://osu.ppy.sh/community/contests/115) until 2021-02-28. There's a lot of work going into this, so you might need the extra time.
+- **2 weeks:** Submissions are judged by separate mapper and musician judging panels.
+- **2 weeks:** Community vote is opened for top 5 overall submissions
+
+Once each of these are completed, the winners will be announced!
+
+## Prizes
+
+No such thing as a contest without prizes! Unlike most contests however, there's a few different way to earn prizes this time around.
+
+2 groups will handle judging: **mappers (including storyboarders!) and musicians.** They'll be expected to evaluate the full package — the remixed song + beatmap + storyboard/video combo. 
+
+Each group has its own specialty of course, so we'll be rewarding prizes to the all members of the highest scoring team from each panel.
+
+- **Mapper panel**
+    - **Runner-up:** 3 months of osu!supporter 
+    - **Winner:**  6 months of osu!supporter + *Mappers' Pick* profile badge
+
+![Mappers' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_mapperspick@2x.png)
+
+- **Musician panel**
+    - **Runner-up:** 3 months of osu!supporter 
+    - **Winner:**  6 months of osu!supporter + *Musicians' Pick* profile badge
+
+![Musicians' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_musicianspick@2x.png)
+
+The legend himself, *Kuba Oms*, will be part of this panel!
+
+That's not all though! When panel results are in, we'll combine them to form the overall top 5 entries. These will be open for a *community vote*, meaning you can pick your favorite entry!
+
+Once the vote is finalized, we'll combine panel scores with community vote scores to determine the overall winner (weighting for each category is undecided).
+
+This winner will be announced during a **live stream showcase of the finalists** with commentary from some of the judges!
+
+- **Overall prizes**
+    - **Runner-up:** 3 months of osu!supporter + *Overall runner-up* profile badge
+    - **Winner:**
+        - 3 months of osu!supporter
+        - *Overall winner* profile badge
+        - *Elite Mapper*, *Elite Storyboarder*, and *Elite Remixer* user titles
```

Comment:
"osu!mixer" was previously used for a musician title. if "Elite Remixer" isn't specifically important i recommend using "osu!mixer" again